doc/languages-frameworks/python.section.md

```diff
@@ -538,6 +538,95 @@ buildPythonPackage rec {
 ```
 Note also the line `doCheck = false;`, we explicitly disabled running the test-suite.
 
+#### Testing Python Packages
+
+It is highly encouraged to have testing as part of the package build. This
+helps to avoid situations where the package was able to build and install,
+but is not usable at runtime. Currently, all packages will use the `test`
+command provided by the setup.py. However, this is currently deprecated
+https://github.com/pypa/setuptools/pull/1878 and your package should provide
+it's own checkPhase.
+
+*NOTE:* The `checkPhase` for python maps to the `installCheckPhase` on a
+normal derivation. This is due to many python packages not behaving well
+to the pre-installed version of the package. Version info, and natively
+compiled extensions generally only exist in the install directory, and
+thus can cause issues when a test suite asserts on that behavior.
+
+*NOTE:* Tests should only be disabled if they don't agree with nix,
+failing tests can still be a good indication that the package is not
+in a valid state.
```

Comment:
```suggestion
in a valid state. At the very least, using ``pythonImportsCheck = [ "packagename" ];`` helps
check that your package is minimally working.
```

Reply:
I forgot about this. Thanks for reminding me